backend/file-system/src/main/java/com/htec/filesystem/service/FileService.java

```diff
@@ -129,4 +132,19 @@ public void saveFileIntoDB(String filePath, String fileName, long shelfId, long
         fileEntity.setCreatedAt(LocalDateTime.now());
         fileRepository.save(fileEntity);
     }
+
+    @Transactional
+    public void updateDeletedFile(AuthUser user, List<Long> fileIds, boolean delete) {
```

Comment:
method name? we are soft-deleting multiple files according to the input parameters

Reply:
Changed to updateDeletedMultipleFiles